src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_3_layout.html.twig

```diff
@@ -321,7 +321,7 @@
 {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? (translation_domain is same as(false) ? label : label) -}}
+            {{- label is not same as(false) ? (translation_domain is same as(false) ? label|raw : label|raw) -}}
```

Comment:
As Symfony forms are an internal feature, it's not a security issue (this block generates the option label block of a form).